.github/workflows/generate_release_note.yml

```diff
@@ -0,0 +1,36 @@
+name: Auto Release Notes
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Instead of creating a new job, can we use the existing one that pushes the artifacts to artifactory?